invenio_rdm_records/marshmallow/json.py

```diff
@@ -28,11 +28,32 @@ class CommunitySchemaV1(BaseSchema):
     secondary = fields.List(SanitizedUnicode())
 
 
-class IdentifierSchemaV1(BaseSchema):
-    """Extra/Alternate identifiers of the record."""
+# 'Fake' Identifiers Field
+def _not_blank(error_msg):
+    """Returns a non-blank validation rule with custom error message."""
+    return validate.Length(min=1, error=error_msg)
 
-    identifier = SanitizedUnicode(required=True)
-    scheme = SanitizedUnicode(required=True)
+
+def Identifiers():
+    """Returns a "fake" Identifiers field.
+
+    Field expects:
+
+        "<scheme1>": "<identifier1>",
+        ...
+        "<schemeN>": "<identifierN>"
+    """
+    return fields.Dict(
+        # scheme
+        keys=SanitizedUnicode(
```

Comment:
Minor: We should later (i.e. not blocking) ensure that the scheme is auto-detected via idutils library.

Reply:
Yes and we have an issue for that: #51